datalake/src/main/java/com/sequenceiq/datalake/flow/stop/handler/SdxStopAllDatahubHandler.java

```diff
@@ -0,0 +1,69 @@
+package com.sequenceiq.datalake.flow.stop.handler;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.dyngr.exception.PollerException;
+import com.dyngr.exception.PollerStoppedException;
+import com.dyngr.exception.UserBreakException;
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.logger.MDCBuilder;
+import com.sequenceiq.datalake.flow.SdxEvent;
+import com.sequenceiq.datalake.flow.start.event.SdxStartFailedEvent;
+import com.sequenceiq.datalake.flow.stop.SdxStopEvent;
+import com.sequenceiq.datalake.flow.stop.event.SdxStopAllDatahubRequest;
+import com.sequenceiq.datalake.service.sdx.stop.SdxStopService;
+import com.sequenceiq.flow.reactor.api.handler.EventHandler;
+
+import reactor.bus.Event;
+import reactor.bus.EventBus;
+
+@Component
+public class SdxStopAllDatahubHandler implements EventHandler<SdxStopAllDatahubRequest> {
+
+    public static final int SLEEP_TIME_IN_SEC = 20;
+
+    public static final int DURATION_IN_MINUTES = 40;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SdxStopAllDatahubHandler.class);
+
+    @Inject
+    private EventBus eventBus;
+
+    @Inject
+    private SdxStopService sdxStopService;
+
+    @Override
+    public String selector() {
+        return "SdxStopAllDatahubRequest";
```

Comment:
this is fishy
shouldn't it come from an Enum somewhere?

Reply:
Unfortunately, the selectors are the weak point of the flow2. If I use another enum It will make more unreadable the flow config and events

Reply:
In the Environment service for example for the env creation flow we use an enum for this, and it does not make it more unreadable at all. On the contrary, it is more robust. It is used in both the handlers and in the actions class.
Can you please adopt this pattern?
See `EnvCreationHandlerSelectors` and `EnvDeleteHandlerSelectors`

You also did the same in this pull request with `EnvStopHandlerSelectors`.